test/e2e/framework.go

```diff
@@ -428,16 +430,20 @@ func (data *TestData) deployAntreaFlowExporter(ipfixCollector string) error {
 func (data *TestData) deployFlowAggregator(ipfixCollector string) error {
 	rc, _, _, err := provider.RunCommandOnNode(masterNodeName(), fmt.Sprintf("kubectl apply -f %s", flowaggregatorYML))
 	if err != nil || rc != 0 {
-		return fmt.Errorf("error when deploying flow aggregator; is %s available on the master Node?", flowaggregatorYML)
+		return fmt.Errorf("error when deploying flow aggregator; %s not available on the master Node", flowaggregatorYML)
 	}
-	err = data.mutateFlowAggregatorConfigMap(ipfixCollector)
-	if err != nil {
+	if err = data.moveDeploymentToSpecificNode(flowAggregatorDeployment, flowAggregatorNamespace, masterNodeName()); err != nil {
+		return err
+	}
```

Comment:
I don't think this is the right way to go:
* if https://github.com/vmware-tanzu/antrea/issues/897 is the root cause of the issue, then we should just skip the affected tests until this issue is resolved, while making sure that all the tests are run in Jenkins
* if there is an extra DNS issue, on top of that, then we can deploy the Aggregator first and access the service through the ClusterIP in Kind tests

Reply:
I think the second point is a good suggestion. Let me quickly try that out instead of skipping.